examples/tutorial/exercises/Exercise-4-dynamic-interactions.ipynb

```diff
@@ -70,26 +69,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Scatter [width=600 height=400 logy=True tools=['box_select'] color_index='cut']\n",
-    "%%opts Scatter (size=1.5 cmap='tab20c')\n",
-    "\n",
+    "scatter_opts = opts.Scatter(width=600, height=400, logy=True, tools=['box_select'], \n",
+    "                            color=dim('cut'), size=1.5, cmap='tab20c')\n",
     "scatter = hv.Scatter(diamonds.sample(10000), 'carat', ['price', 'cut', 'clarity']).select(carat=(0, 3))\n",
     "boxwhisker = hv.BoxWhisker(scatter, 'clarity', 'price')\n",
     "\n",
-    "scatter + boxwhisker"
+    "scatter.opts(scatter_opts) + boxwhisker"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By default, the ``BoxWhisker`` element here will statically display the whole distribution. But if you try out the \"Box select\" tool, you can select a subset of the Scatter points.  Can we link the boxwhisker plot to selections made on the ``Scatter`` plot, so that we can see distributions in that particular region of the data space? Yes, as long as we have these three things:\n",
+    "By default, the<code>BoxWhisker</code> element here will statically display the whole distribution. But if you try out the \"Box select\" tool, you can select a subset of the Scatter points.  Can we link the boxwhisker plot to selections made on the<code>Scatter</code> plot, so that we can see distributions in that particular region of the data space? Yes, as long as we have these three things:\n",
```

Comment:
```suggestion
    "By default, the <code>BoxWhisker</code> element here will statically display the whole distribution. But if you try out the \"Box select\" tool, you can select a subset of the Scatter points.  Can we link the boxwhisker plot to selections made on the <code>Scatter</code> plot, so that we can see distributions in that particular region of the data space? Yes, as long as we have these three things:\n",
```